public/views/index.html

```diff
@@ -300,3 +296,159 @@ <h3>XOXO</h3>
 </section>
 <!-- Footer End -->
 </div>
+
+<!-- New landing page -->
+<!-- 
```

Comment:
Let's get rid of this large chunk of commented out code below @clintfidel @saheedt